crates/client/sync/src/commitments/lib.rs

```diff
@@ -205,34 +213,52 @@ pub fn update_state_root(
 fn contract_trie_root(
     csd: &CommitmentStateDiff,
     overrides: Arc<OverrideHandle<Block<Header<u32, BlakeTwo256>, OpaqueExtrinsic>>>,
-    mut bonsai_contract: MutexGuard<BonsaiStorage<BasicId, BonsaiDb, Pedersen>>,
-    bonsai_contract_storage: Arc<Mutex<BonsaiStorage<BasicId, BonsaiDb, Pedersen>>>,
+    bonsai_contract: &mut BonsaiStorage<BasicId, BonsaiDb, Pedersen>,
+    bonsai_contract_storage: &mut BonsaiStorage<BasicId, BonsaiDb, Pedersen>,
     block_number: u64,
     maybe_block_hash: Option<H256>,
 ) -> Result<Felt252Wrapper, BonsaiStorageError<BonsaiDbError>> {
     let identifier = bonsai_identifier::CONTRACT;
     bonsai_contract.init_tree(identifier)?;
 
+    let start1 = std::time::Instant::now();
```

Comment:
Is it really a good idea to keep timers in the codebase? Just wondering, open to feedback :)

Reply:
i think they should go once i get perf and flamegraph working again
I left them here because adding/removing them between every commit was bothering me